src/main/java/com/refinedmods/refinedstorage/inventory/item/validator/PatternItemValidator.java

```diff
@@ -15,6 +17,13 @@ public PatternItemValidator(World world) {
 
     @Override
     public boolean test(ItemStack stack) {
+        return test(world, stack);
+    }
+
+    public static boolean test(World world, ItemStack stack) {
+        if (stack.getItem() == RSItems.PATTERN.get()) {
```

Comment:
Why can't this check be done from the original `test` method?

Reply:
I first added the check and after that I made an impulsive refactor to move body of the test method to a static method, so that the whole check can be called from two other places without creating a throwaway instance, that's it.
I can revert the refactor part if you want

Reply:
Please do